apps/mobile/src/contexts/MobileAnalyticsProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import { Mixpanel } from 'mixpanel-react-native';
+import { PropsWithChildren } from 'react';
+
+import AnalyticsProvider, {
+  IdentifyFunction,
+  TrackFunction,
+} from '~/shared/contexts/AnalyticsContext';
+
+let instance: Mixpanel | undefined = undefined;
+
+// TODO env
+const token = '39358d4e4b8aeb0ee10e4ddbcb7a53eb';
+const apiUrl = 'https://analytics.gallery.so';
```

Comment:
did you mean to handle this TODO in a later PR?

Reply:
yeah I added some more of these around the codebase. just so I have something to grep when we finally support multiple environments